probe-rs/src/probe/jlink/mod.rs

```diff
@@ -826,3 +884,15 @@ impl From<jaylink::Error> for DebugProbeError {
         DebugProbeError::ProbeSpecific(Box::new(e))
     }
 }
+
+impl TryFrom<jaylink::Interface> for WireProtocol {
+    type Error = DebugProbeError;
+
+    fn try_from(interface: Interface) -> Result<Self, Self::Error> {
+        match interface {
+            Interface::Jtag => Ok(WireProtocol::Jtag),
+            Interface::Swd => Ok(WireProtocol::Swd),
+            unknown => todo!("Return a proper error here!"),
```

Comment:
Fix this warning please :)

Also I think we should return an Error here. I would rather not get any todos upstream. In the future when we add another protocol (I think this is unlikely, but then we also don't need to make this enum non-exhaustive), this will cause random crashes to users ...

Reply:
This is not an issue when we add new protocols, but when `jaylink` adds a new protocol.

I will remove it, didn't intend to commit it like this.